packages/nodes-base/nodes/Lemlist/descriptions/UnsubscribeDescription.ts

```diff
@@ -13,14 +13,17 @@ export const unsubscribeOperations: INodeProperties[] = [
 			{
 				name: 'Add',
 				value: 'add',
+				action: 'Add an unsubscribe',
```

Comment:
Unsubscribe an email?

Reply:
Changed to `Add an email to an unsubscribe list` to follow the pattern.